ts/compiler.ts

```diff
@@ -1,25 +1,38 @@
-import * as ejs from './index'
-import * as fs from 'fs'
-import * as version from './version'
+import * as ejs from './index';
+import * as fs from 'fs';
+import * as version from './version';
 
-if (process.argv.length < 1) {
-  console.error('Usage: ejs input.js [output.js]');
+if (process.argv.length < 3) {
+  console.error('Usage: node compiler.js input.js');
   process.exit(1);
 }
-const input = process.argv[0];
-const output = process.argv[1];
-
-function log(str: string) {
-  console.log(str);
-}
+const input = process.argv[2];
 
 try {
-  const code = fs.readFileSync(input);
-  const opts = {
-    consoleLog: log,
-    version: () => { console.log(version.EJSVERSION); }
-  };
-  const result = ejs.compile(code.toString(), opts);
+  const code = fs.readFileSync(input),
+        opts = {
+          consoleLog: (str: string) => { console.log(str); },
+          version: () => { console.log(version.EJSVERSION); },
+          whitelistCode: {
```

Comment:
Should we remove this whitelist? I think it was only added for debugging.

In fact, perhaps we should remove this file entirely. I don't think it is needed. If you found it useful, then let's keep it.

Reply:
I say keep it for now; @lchsam & I cleaned it up so it is usable.
We can always remove it later.